src/pages/form/step-form/components/Step1/index.tsx

```diff
@@ -0,0 +1,125 @@
+import { Form } from '@ant-design/compatible';
+import { Button, Divider, Input, Select } from 'antd';
+import React, { Fragment } from 'react';
+
+import { Dispatch } from 'redux';
+import { FormComponentProps } from 'antd/es/form';
+import { connect } from 'dva';
+import { StateType } from '../../model';
+import styles from './index.less';
+
+const { Option } = Select;
+
+const formItemLayout = {
+  labelCol: {
+    span: 5,
+  },
+  wrapperCol: {
+    span: 19,
+  },
+};
+interface Step1Props extends FormComponentProps {
+  data?: StateType['step'];
+  dispatch?: Dispatch<any>;
+}
+
+const Step1: React.FC<Step1Props> = props => {
+  const { form, dispatch, data } = props;
+  if (!data) {
+    return null;
+  }
+  const { getFieldDecorator, validateFields } = form;
+  const onValidateForm = () => {
+    validateFields((err: any, values: StateType['step']) => {
+      if (!err && dispatch) {
+        dispatch({
+          type: 'formAndstepForm/saveStepFormData',
+          payload: values,
+        });
+        dispatch({
+          type: 'formAndstepForm/saveCurrentStep',
+          payload: 'confirm',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'payload' is not sorted alphabetically](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)